com.unity.multiplayer.mlapi/Tests/Editor/BitStreamTests.cs

```diff
@@ -794,5 +794,21 @@ public void TestStringPackedDiff()
             Assert.That(readBuilder.ToString(), Is.EqualTo(testString));
             Assert.That(stringCompare.ToString(), Is.EqualTo(testString));
         }
+
+        [Test]
+        public void TestSerializationPipelineBool()
+        {
+            NetworkBuffer buffer = new NetworkBuffer();
+            NetworkWriter writer = new NetworkWriter(buffer);
+            NetworkReader reader = new NetworkReader(buffer);
```

Comment:
```suggestion
            var buffer = new NetworkBuffer();
            var writer = new NetworkWriter(buffer);
            var reader = new NetworkReader(buffer);
```

I'd suggest using `var` when it's too obvious (in this case, types are literally on the right side)